### PR TITLE
Remove unused fields

### PR DESCRIPTION
**Pull request by jpbruinsslot**
Removal of unused fields in scheduler models

### CODE REVIEW

File: mula/scheduler/models/boefje.py [hunk 1]
```diff
@@ -23,7 +17,8 @@ class BoefjeMeta(BaseModel):
     id: str
     boefje: Boefje
     input_ooi: Optional[str]
-    arguments: Dict[str, Any]
```
Comment (ammar92) [inline mula/scheduler/models/boefje.py:26]: This is actually a nice hint, unless the key type is changed?

File: mula/scheduler/models/boefje.py [hunk 2]
```diff
@@ -23,7 +17,8 @@ class BoefjeMeta(BaseModel):
...
+    arguments: Dict = {}
```
Comment (ammar92) [inline mula/scheduler/models/boefje.py:20]: Mutable type, use a `Field` with `default_factory` argument

**Review by ammar92 — COMMENTED** [review]
Just one remark (and one question)

File: mula/scheduler/models/boefje.py (hunk 1 above)
Comment (jpbruinsslot) [reply mula/scheduler/models/boefje.py:26]: I changed it to resemble the same structure in boefjes. However I'm ok with reverting it, we should consider changing it to both to the correct form.

File: mula/scheduler/models/boefje.py [hunk 3]
```diff
@@ -23,7 +17,8 @@ class BoefjeMeta(BaseModel):
     id: str
     boefje: Boefje
     input_ooi: Optional[str]
-    arguments: Dict[str, Any]
+    arguments: Dict[str, Any] = Field(default=dict)
```
Comment (ammar92) [inline mula/scheduler/models/boefje.py:20]:
```suggestion
    arguments: Dict[str, Any] = Field(default_factory=dict)
```

**Review by ammar92 — APPROVED** [review]

**Review by Donnype — APPROVED** [review]

**Review by underdarknl — APPROVED** [review]

**Pull request merged**